gulp-utils.js

```diff
@@ -31,9 +21,25 @@ function getAsc() {
   }
 
   asc = require("assemblyscript/bin/asc");
-
+  
   const fs = require("fs");
   const pathModule = require("path");
+
+  // Create parent directories if they don't exist
+  function mkdirp(path){
```

Comment:
it doesn't really work like `mkdir -p` (e.g. if you pass `foo/bar.txt` you'd expect `bar.txt` directory created by `mkdir-p`), so either logic or name should be changed